schedule/yam/test_cases/skip_registration.yaml

```diff
@@ -9,17 +10,23 @@ schedule:
   registration:
     - installation/registration/skip_registration
   extension_module_selection:
-    - installation/module_selection/select_module_desktop
+    - installation/module_selection/skip_module_selection
```

Comment:
we have small test coverage for full medium, why to change the desktop here? I would prefer to keep it, pvm can be installed with a desktop (although the booting will be in textmode).

Reply:
Solved here d0428107a9ea77bd0b32fde3832efb574fb8a781

Thanks